File: python/sprint1_nonfinals/b.py
def check_parity(a: int, b: int, c: int) -> bool:
    # Здесь реализация вашего решения
    pass

def print_result(result: bool) -> None:
    if result:
        print("WIN")
    else:
        print("FAIL")

a, b, c = map(int, input().strip().split())
print_result(check_parity(a, b, c))

File: swift/sprint1_nonfinals/b.swift
func checkParity(_ a : Int, _ b : Int, _ c : Int) ->Bool {
    // Здесь реализация вашего решения
}

func readInts() -> [Int] {
    return readLine()!.split(separator: " ").compactMap { Int($0)! }
}

func print(_ result: Bool) {
    print(result ? "WIN" : "FAIL")
}

func main() {
    let input = readInts()
    print(checkParity(input[0], input[1], input[2]))
}

main()
